### ISSUE TITLE
Fix jsx-a11y/no-noninteractive-element-interactions warnings raised by jsx-a11y linter.

### ISSUE THREAD

**Issue by yzen** [firefox-devtools/debugger#7540]
To get a list of accessibility warnings raised by jsx-a11y linter run: `yarn lint:jsx-a11y`.

From that list please note all instances that are flagged to have "jsx-a11y/no-noninteractive-element-interactions" problem. Each one needs to be addressed individually.

Once all of the instances are resolved make sure to change this setting in `.eslintrc.jsx-a11y` from:
```
"jsx-a11y/no-noninteractive-element-interactions": ["warn", {
  "handlers": ["onClick", "onError", "onLoad", "onMouseDown", "onMouseUp", "onKeyPress", "onKeyDown", "onKeyUp"],
  "body": ["onError", "onLoad"],
  "iframe": ["onError", "onLoad"],
  "img": ["onError", "onLoad"]
}],
```

to
```
"jsx-a11y/no-noninteractive-element-interactions": ["error", {
  "handlers": ["onClick", "onError", "onLoad", "onMouseDown", "onMouseUp", "onKeyPress", "onKeyDown", "onKeyUp"],
  "body": ["onError", "onLoad"],
  "iframe": ["onError", "onLoad"],
  "img": ["onError", "onLoad"]
}],
```

**Comment by Garbee** [comment]
The [error in question](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-interactions.md) once again revolves around using generic elements instead of buttons for actions. Through solving #7537 I think most of the errors here, if not all, should be resolved as well.

**Comment by jasonLaster** [comment]
How's it going @codingnninja?

**Comment by janodvarko** [comment]
Firefox Developer Tools (including Debugger tool) development is entirely happening in Bugzilla and new issues should be reported here: https://bugzilla.mozilla.org/enter_bug.cgi?product=DevTools&component=Debugger

Also, supporting JSX in the code base is not on our long term roadmap, so this might be invalid report.

Honza
